localscripts/generate-po2json.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+[ -x "$(command -v npm)" ] || { >&2 echo "npm is required but not installed, exiting."; exit 1; }
+
+echo ""
+echo "Installing po2json..."
+echo ""
+npm install po2json
+
+echo ""
+echo "Generating .po to .json files..."
+echo ""
+
+pofilenames=$(ls ./translations/planet4-plugin-gutenberg-blocks/languages/*.po)
+for eachpofile in $pofilenames
+do
+   subdomain='planet4-blocks-backend'
+   if [[ "$eachpofile" =~ .*"$subdomain".* ]]; then
+     # Generate json file from .po file. (Note: The o/p filename should be ${domain}-${locale}-${handle}.json)
+     npx po2json "$eachpofile" "${eachpofile/.po/}-planet4-blocks-script.json" -f jed1.x;
+   else
+     npx po2json "$eachpofile" "${eachpofile/.po/}-planet4-blocks-frontend.json" -f jed1.x;
```

Comment:
You can assign the suffix to a variable in the if statement. Then we don't need to duplicate the rest of the `command`
```
if [[ "$eachpofile" =~ .*"$backend_domain".* ]]; then
  suffix=backend
else
  suffix=frontend
fi
npx po2json "$eachpofile" "${eachpofile/.po/}-planet4-blocks-${suffix}.json" -f jed1.x;
```